tools/topology/topology1/sof-rn-rt5682-rt1019.m4

```diff
@@ -0,0 +1,65 @@
+#
+# Topology for Renoir with I2S SP and DMIC.
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`acp-sp.m4')
+include(`acp-dmic.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include ACP DSP configuration
+include(`platform/amd/acp.m4')
+
+# Playback pipeline 1 on PCM 0 using max 2 channels of s16le.
+# Schedule 96 frames per 2000us deadline on core 0 with priority 0
```

Comment:
is 2ms required? IIRC all existing topologies assume a 1ms period.

Reply:
These are just sample topologies to show case usage of AMD platforms, nothing specific about 2ms.